GitUI/FilterBranchHelper.cs

```diff
@@ -51,10 +50,9 @@ public FilterBranchHelper(ToolStripComboBox toolStripBranches, ToolStripDropDown
             : this()
         {
             _NO_TRANSLATE_toolStripBranches = toolStripBranches;
-            _NO_TRANSLATE_toolStripDropDownButton2 = toolStripDropDownButton2;
```

Comment:
💭 It's possible this field had special meaning in the translation tools?

Reply:
Yes, all other fields are picked up and are exported to Transifex

Reply:
I thought the `_NO_TRANSLATE_` prefix was an indicator that such fields are ignored by the translation system and as such removing them has no impact.

Reply:
Clarification: All other strings are propagated to translation, _NO_TRANSLATE strings are ignored. Anyway, strings that are not needed should be removed.

I do not see why the button was copied in the first place, this is a change that should be checked before merged.